include/fuzzuf/algorithms/afl/afl_state.hpp

```diff
@@ -61,7 +61,7 @@ struct AFLStateTemplate {
   explicit AFLStateTemplate(
       std::shared_ptr<const AFLSetting> setting,
       std::shared_ptr<executor::AFLExecutorInterface> executor,
-      std::unique_ptr<optimizer::Optimizer<u32>> &&mutop_optimizer);
+      std::shared_ptr<optimizer::Optimizer<u32>> &&mutop_optimizer);
```

Comment:
The same as https://github.com/fuzzuf/fuzzuf/pull/84/files#r935112110

Reply:
fixed in 6b50cebeecd293f89664bc2a8165c3b89bc7ff35